packages/fast-component-explorer/app/index.tsx

```diff
@@ -11,10 +14,30 @@ const root: HTMLElement = document.createElement("div");
 root.setAttribute("id", "root");
 document.body.appendChild(root);
 
-const DragDropApp: any = DragDropContext(HTML5Backend)(App);
+/**
+ * Create Application Insights SDK for Telemetry
+ * https://docs.microsoft.com/en-us/azure/azure-monitor/learn/nodejs-quick-start
+ */
+const browserHistory = createBrowserHistory({ basename: "" });
+var reactPlugin = new ReactPlugin();
+var appInsights = new ApplicationInsights({
+    config: {
+        instrumentationKey: "ecda4203-b38b-42ad-b181-f0f7f2b1d8fa",
```

Comment:
I understand this isn't a security threat, leaving this in source for the world to see opens the opportunity for this to get stolen and misused.

We need to tokenize this as part of a build step so private information stays private.

Reply:
@nicholasrice Please read https://devblogs.microsoft.com/premier-developer/alternative-way-to-protect-your-application-insights-instrumentation-key-in-javascript/ and then let me know how you'd like to proceed.

Reply:
*Added this to the original comment thread*

@nicholasrice I agree, it leaves us open to data being passed that does not originate from the relevant sites.

I would like to see:

Tokens used in place of generated keys
Tokens for production and staging
Factoring out the logic for use in both the explore site and the doc site if this is expected to happen in the future

Reply:
My understanding is that, with this implementation, anyone that comes across this key could pollute our data - is that correct?

Reply:
My understanding is that, with this implementation, anyone that comes across this key could pollute our data - is that correct?